WikipediaUnitTests/Code/URLParsingAndRoutingTests.swift

```diff
@@ -94,4 +94,25 @@ class URLParsingAndRoutingTests: XCTestCase {
         XCTAssertEqual(url.resolvingRelativeWikiHref(".//dev/null")?.absoluteString, "https://en.wikipedia.org/wiki//dev/null")
     }
     
+    /// Ensure the 'transcoded' path component is inserted after the language path component for audio hosted on Wikipedia
+    func testWikipediaCompatibilityAdjustments() {
+        let url = URL(string: "https://upload.wikimedia.org/wikipedia/en/3/3f/DeanScream.ogg")!
```

Comment:
Oh goodness, I've now officially listened to this scream more times than a human should.